tests/collector/integration.js

```diff
@@ -734,10 +740,16 @@ describe('tests/collector/integration.js >', function () {
       .then(() => u.awaitHeartbeat())
       .then(() => u.postGenerator(gen4))
 
-      .then(() =>
-        u.awaitBulkUpsert()
-        .should.eventually.be.rejectedWith(Promise.TimeoutError)
-      );
+      .then(() => u.awaitBulkUpsert())
+      .then(({ req }) => {
+        console.log(req.body);
+        expect(req.body).to.deep.equal([{
+          name: 'sub1|asp1',
+          messageCode: 'ERROR',
+          messageBody: 'Unauthorized (http://www.example.com)',
```

Comment:
A more detailed error message would be helpful, so it's clear that it's a problem with the OAuth endpoint rather than the data source.